# training parameters
train:
  epochs: 1
  lr: 0.001              
  seed: 42               
  cudnn: True
  classes: 3
  random_state: 420      
  valid_size: 0.1        
  batch_size: 3          
  num_workers: 2         
  w_decay: 0.01              
  optim_factor: 0.25
  optim_patience: 2
  eps: 1e-08
  amsgrad: False
  betas_min: 0.9
  betas_max: 0.999
  logdir: "content/full_model2"
  minimize_metric: True
  fp16: null
  verbose: True

# testing parameters
test:
  batch_size: 3
  shuffle: False
  num_workers: 4

# encoder parameters
encoder:
  name: "efficientnet-b3"
  weight: "imagenet"
  activation: null